packages/hashi/src/mediator.js

```diff
@@ -0,0 +1,88 @@
+/*
+ * This class manages all message listening and sending from the postMessage
+ * layer. All interfaces that need to message via the postMessage layer should
+ * do so via an instance of this class.
+ */
+
+class Mediator {
```

Comment:
Mediators++! :)

Reply:
Fun fact, during this process I discovered that Vue devtools uses `window.postMessage` for its messaging!